backend/bot/index.js

```diff
@@ -1,6 +1,21 @@
-var SlackBot = require('slackbots');
+const SlackBot = require('slackbots');
 
-var bot = new SlackBot({
+const storage = require('../storage/storage');
```

Comment:
Plz don't access `storage` module directly from here, fetch players from the `match-repository` in the `recordMatch` method and pass it to the `bot` module in a method argument.